sdk/client/const.go

```diff
@@ -6,4 +6,6 @@ const (
 	AnnotationEnv = "NS_NETWORKSERVICEMESH_IO"
 	// NSMConnectRetries - Specify how many time we could retry to do a connection retrying.
 	NSMConnectRetries = "NSM_CLIENT_CONNECT_RETRIES"
+	// NamespaceEnv - NamespaceEnv is the namespace a container is running in
+	NamespaceEnv = "NSM_NAMESPACE"
```

Comment:
This constant is in several places?

Reply:
There is second in the admission-webhook, to skip depending on sdk. It's used only for exposing the namespace to the client labels (in order to be exposed to the crossconnect-monitor)